src/index.js

```diff
@@ -67,6 +68,7 @@ async function startClientBot(client) {
 
 function handleCriticalError(error) {
   saveErrorLog(error);
+  registerGlobalErrorHandlers(client);
```

Comment:
pass error to registerGlobalError

```suggestion
  registerGlobalErrorHandlers(client, error);
```